src/index.html

```diff
@@ -1,16 +1,31 @@
 <!DOCTYPE html>
 <html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>tv-maze</title>
+  </head>
 
-<head>
-    <meta charset="UTF-8">
-    <meta http-equiv="X-UA-Compatible" content="IE=edge">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>Leaderboard Game</title>
-</head>
-
-<body>
+  <body>
+    <header>
+      <a href="./index.html"><img src="#" alt="logo" class="logo" /></a>
+      <ul class="menu">
+        <li class="active">
+          <a href="">Shows </a> <span class="show-count"></span>
+        </li>
+        <li><a href="">Movies</a></li>
+        <li><a href="">Events</a></li>
+      </ul>
+    </header>
```

Comment:
- [ ] For better lock and feel, I would recommend you to put the header fixed to the top and add a height of the header as well.